lib/browser.ts

```diff
@@ -188,7 +195,21 @@ export class ProtractorBrowser extends Webdriver {
    *
    * @type {boolean}
    */
-  ignoreSynchronization: boolean;
+  set ignoreSynchronization(value) {
```

Comment:
I think we should take this opportunity to improve the naming here.  I think we had discussed `waitForAngularEnabled(true/false)`